### PR TITLE
[Papercut] [SW-15773] Fix snippets multi edit order

### PR DESCRIPTION
**Pull request by SimonBaeumer**
## Description

Please describe your pull request:
- Why is it necessary?
  - E-Mail status was not correctly displayed
- What does it improve?
  - Usability of the multi edit module for orders
- Does it have side effects?
  - No

| Questions | Answers |
| --- | --- |
| BC breaks? | no |
| Tests pass? | yes |
| Related tickets? | [SW-15773](https://issues.shopware.com/#/issues/SW-15773) |
| How to test? | Bulk change orders with and without documentation generation |

### CODE REVIEW

**Comment by janbuecker** [comment]
Please fix your code styling. :-)

File: themes/Backend/ExtJs/backend/order/store/batch.js [hunk 1]
```diff
@@ -39,11 +39,13 @@ Ext.define('Shopware.apps.Order.store.Batch', {
      * @string
      */
     extend:'Ext.data.Store',
+
```
Comment (janbuecker) [inline themes/Backend/ExtJs/backend/order/store/batch.js]: please revert this changes since they just bloat the commit

File: themes/Backend/ExtJs/backend/order/controller/batch.js [hunk 2]
```diff
@@ -215,6 +215,7 @@ Ext.define('Shopware.apps.Order.controller.Batch', {
             settingValues = me.getSettingsPanel().getValues(),
             nextIndex = index + 1,
             percentage = nextIndex / orders.length,
+            results = [],
```
Comment (janbuecker) [inline themes/Backend/ExtJs/backend/order/controller/batch.js]: this is unused

Comment (SimonBaeumer) [reply themes/Backend/ExtJs/backend/order/controller/batch.js]: Fixed ;)

File: themes/Backend/ExtJs/backend/order/store/batch.js (hunk 1 above)
Comment (SimonBaeumer) [reply themes/Backend/ExtJs/backend/order/store/batch.js]: Fixed

**Comment by janbuecker** [comment]
Thanks @SimonBaeumer 👍
